src/app/constants.ts

```diff
@@ -43,3 +47,4 @@ export const GRID_COLUMNS = {
 // this REGEXP adds the requirement of ending with a TLD as defined on RFC2396
 export const TLD_REGEXP = /^.*\.([a-zA-Z\-])([a-zA-Z\-]{0,61})([a-zA-Z\-])$/
 export const ORCID_REGEXP = /(\d{4}-){3,}\d{3}[\dX]$/i
+export const ORCID_URI_REGEXP = /(http[s]?:\/\/)(orcid\.org\/|qa\.orcid\.org\/|dev\.orcid\.org\/|localhost.*)(\d{4}-){3,}\d{3}[\dX]$/
```

Comment:
Perfect! 

I am going to create another issue to implement this on the search as well, and let's keep reusing/improving this reg for URLs. Just two changes here: 

a- please add support to sandbox
b- I am adding a testing URL as a comment for each regexp on the `feature/register` PR, look:
https://github.com/ORCID/orcid-angular/blob/938b63671f7aec35bbf657353193fd97ab606104/src/app/constants.ts#L42-L55
So please add a URL like this one https://regex101.com/r/6U0wA9/1/ (with sandbox) as a comment on top of you regexp

Reply:
@leomendoza123 Ok. Sounds great!